ajax/libs/gzip-js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "gzip-js",
+  "description": "GZIP in pure JavaScript (works in the browser)",
+  "filename": "gzip.min.js",
+  "keywords": [
+    "gzip",
+    "zip",
+    "pure javascript"
+  ],
+  "author": "T. Jameson Little <t.jameson.little@gmail.com>",
+  "homepage": "https://manifestwebdesign.github.io/angular-gridster",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/beatgammit/gzip-js.git"
+  },  
+  "npmName": "gzip-js",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "*.js"
```

Comment:
Please use `"gzip*"` this pattern, because `"*.js"` will fetch all javascript files, but we don't need all the JavaScript files, we just want to get the files we want.
In this case, we'd like to add `"gzip*"` files which under `lib`, so let use this pattern.
Thanks.